pallets/gear/src/manager.rs

```diff
@@ -63,7 +63,7 @@ where
     T::AccountId: Origin,
 {
     fn collect(&self) -> State {
-        let actors: BTreeMap<ProgramId, ExecutableActor> = PrefixIterator::<H256>::new(
+        let actors = PrefixIterator::<H256>::new(
```

Comment:
I prefer previous code style: 
```suggestion
        let actors: BTreeMap<ProgramId, Option<ExecutableActor>> = PrefixIterator::<H256>::new(
```
and no-generic `collect()` at the end

Reply:
We can replace some types inside `BTreeMap` with underscore but I agree that avoiding turbo-fish is preferable.

Reply:
I'll keep that in mind for the future. Done